### ISSUE TITLE
GUI hangs / slow 

### ISSUE THREAD

**Issue by akis-t** [simulationcraft/simc#4450]
### Describe the bug
Just launch SimulationCraft even with .ini settings zapped. None of the drop down listboxes work, for example cannot choose EU or other regions, or anything else. Clicking on "Simulation" simply freezes the GUI and many other similar issues. Completely unusable.

* SimulationCraft Version (required)
8.01, 8.02

**Comment by scamille** [comment]
For GUI issues, it would really help to get some more information:
What is your operating system?
Graphic card and driver?
Your **dxdiag** file - Instructions here - 
  https://us.battle.net/support/en/article/Obtaining-System-Files

I can't realy promise anything since GUI problems are often somewhat obscure, the GUI library (Qt) used is a very complex and large beast in which we are only users, and our windows release maintainers are usually quite time limited.

**Comment by sondreomar** [comment]
TL;DR: If you have a Nvidia GPU update drivers to latest version (399.07 at time of writing).

I had very similar issues with the GUI on v801 r1. Was about to report I also had this issue, but found that updating Nvidia drivers to version 399.07 fixed it. Just in case this doesn't resolve the issue for others and for clarity's sake, the specific issues I had with the GUI was as follows: 

- Selecting a value from any drop-down listbox didn't change settings (at least visually, settings seemed to change to the selected value after restarting the program). 
- After startup I could select any tab once, but after that all other tabs except the "Welcome", "Help" or "Log" tabs became unresponsive. 
- I did not have these issues with previous versions. 

Specs at the time of experiencing the issues:
- OS: Windows 10 Pro (Build 10.0.17134)
- GPU: GTX 1080 Ti (Driver version: 398.36)

**Comment by akis-t** [comment]
I can confirm that with NVidia drivers 24.21.13.9907 (399.07) those bugs seem to have gone.

As a software developer I am finding it hard to understand how a desktop GUI Windows application could be affected so much by the graphics driver, especially for standard Windows UI elements, like check boxes, radio buttons, drop down listboxes etc.

**Comment by navv1234** [comment]
Qt has been somewhat notorious on these issues (graphics drivers) for a long time. The problem has anecdotally seemed to increase when we swapped from webkit to webengine (chromium-based) html rendering.

**Comment by scamille** [comment]
Closing this since the original problem was resolved with new graphic drivers.
